tests/test_autorift.py

```diff
@@ -73,6 +73,9 @@ def test_golden_products(comparison_environments, job_name, keep):
                         f'    Develop: {develop_jobs}')
 
     for main_job, develop_job in zip(main_jobs, develop_jobs):
+        if main_job.failed() or develop_job.failed():
+            continue
+
```

Comment:
What's the purpose of this `if`?

Reply:
for this test, it just hard crashes if any job fails, but (1) we already set a failure message (reported later) for that case earlier and (2) I'd like it to just compare everything it can and report out on those as well. 

For example, USGS may reprocess a Landsat scene so a job may fail, but I will still want to look at the results for all other 15 comparisons.

Admittedly, this should probably be its own PR -- didn't realize I added it here was well.

Reply:
Sounds reasonable.